core/src/main/java/com/novoda/noplayer/SubtitleView.java

```diff
@@ -0,0 +1,82 @@
+package com.novoda.noplayer;
+
+import android.content.Context;
+import android.graphics.Canvas;
+import android.util.AttributeSet;
+import android.view.View;
+
+import com.novoda.noplayer.model.NoPlayerCue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public final class SubtitleView extends View {
```

Comment:
Can the client then use any view? If yes, can we create an interface with `setCues`?